Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/CorrelationAttribute.java

```diff
@@ -117,11 +113,16 @@ public String getCorrelationValue() {
     }
 
     /**
+     * Set the correlation value.  Requires that the correlation type has already
+     * been set (for data normalization purposes).
+     * 
      * @param correlationValue the correlationValue to set
      */
-    public void setCorrelationValue(String correlationValue) {
-        // Lower-case all values to normalize and improve correlation hits, going forward make sure this makes sense for all correlation types
-        this.correlationValue = correlationValue.toLowerCase();
+    public void setCorrelationValue(String correlationValue) throws CorrelationAttributeNormalizationException {
+        if(this.getCorrelationType() == null){
+            throw new IllegalStateException("Correlation Type must be set before calling setCorrelationValue");
```

Comment:
While this does seem like an illegal state, unchecked exceptions tend to cause trouble. It's fine to use one of the normal CR ones.